### DISCUSSION TITLE
Sveltekit SEO issue - Autogenerating pages with root path.

### DISCUSSION THREAD

**Discussion by pateljoel** [sveltejs/kit#9875] [Q&A]
I'm trying to use Sveltejs for a project that requires my URLs to use this path structure (e.g. $BASE_URL/nikon-z9) rather than this path structure $BASE_URL/tech/nikon-z9.

I don't know if Sveltekit supports this and have tried everything to achieve this structure.

Any help to achieve this would be greatly appreciated

**Comment by kristjanmar** [comment]
Create this file: /routes/nikon-z9/+page.svelte

That should make it accessible under yourdomain.com/nikon-z9

**Comment by pateljoel** [comment]
@kristjanmar  Having a look at this, this doesn't look scalable in the long term for my usecase in autogenerating pages.

Is there a way to make this dynamically generated with a list of slug routes in Sveltekit?

**Comment by pateljoel** [comment]
Any help with this?
